### PR TITLE
Feature/issue 202 response time calculations

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@rowanseymour ready for review

**Comment by miltontony** [comment]
I'm going to add more tests

File: casepro/statistics/signals.py [hunk 1]
```diff
@@ -48,23 +48,27 @@ def record_new_outgoing(sender, instance, created, **kwargs):
                 seconds_since_open = ceil(td.total_seconds())
                 DailySecondTotalCount.record_item(day, seconds_since_open,
                                                   DailySecondTotalCount.TYPE_TILL_REPLIED, org)
-
+            # don't count self-assigned cases
```
Comment (rowanseymour) [inline casepro/statistics/signals.py]: This comment being here is a bit misleading as the check for this is actually line 59

File: casepro/statistics/migrations/0010_existing_case_timings_count.py [hunk 2]
```diff
@@ -60,17 +60,25 @@ def calculate_totals_for_cases(apps, schema_editor):
...
-                start_date = action.created_on
-            except CaseAction.DoesNotExist:
-                start_date = case.opened_on
 
-            td = first_response.created_on - start_date
-            seconds_since_open = ceil(td.total_seconds())
-            DailySecondTotalCount.objects.create(day=day, item_type='A', scope='partner:%d' % partner.pk,
-                                                 count=1, seconds=seconds_since_open)
+            # don't count self-assigned cases
+            if case.assignee == partner:
```
Comment (rowanseymour) [inline casepro/statistics/migrations/0010_existing_case_timings_count.py]: This check is always true - not needed I think

**Comment by miltontony** [comment]
@rowanseymour review please

**Pull request merged**